hudi/README.md

```diff
@@ -0,0 +1,72 @@
+# Agent Check: Hudi
+
+## Overview
+
+This check monitors [Hudi][1].
+It is compatible with Hudi [versions][2] `0.10.0` and above.
+
+## Setup
+
+### Installation
+
+The Hudi check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. [Configure][4] the [JMX Metrics Reporter][5] in Hudi:
+
+    ```
+    hoodie.metrics.on=true
+    hoodie.metrics.reporter.type=JMX
+    hoodie.metrics.jmx.host=<JMX_HOST>
+    hoodie.metrics.jmx.port=<JMX_PORT>
+    ```
+
+
+2. Edit the `hudi.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your hudi performance data.
+   See the [sample hudi.d/conf.yaml][6] for all available configuration options.
+
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated when running the Datadog Agent [status command][12].
+   You can specify the metrics you are interested in by editing the [configuration][6].
+   To learn how to customize the metrics to collect visit the [JMX Checks documentation][7] for more detailed instructions.
+   If you need to monitor more metrics, contact [Datadog support][8].
+
+3. [Restart the Agent][9]
+
+
+### Validation
+
+[Run the Agent's `status` subcommand][10] and look for `hudi` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+Hudi does not include any metrics.
```

Comment:
Should this be changed to say Hudi does have metrics?

Reply:
Yep, good catch!